atlp-rwanda/atlp-devpulse-bn
# Ft notifications tickets cycle

### PR Description

This PR allows users to receive in-app notifications and email notifications about their tickets and changes in their application stage.

### Description of tasks that were expected to be completed

As an applicant I want to receive both in-app and email notifications when an action has been taken on my application. 

User is notified when: 
- There are new replies on tickets they have created 
- A score has been added to their technical test
- A score has been added to their interview test
- They have been accepted or rejected from the application cycle


### How has this been tested?
- Clone repo
- Checkout branch `ft-notifications-tickets-cycle`
- Make sure that you set the environment variables referring to `.env.example`
- Run `npm install`, then `npm run dev` to check if the app is running successfully


### Track PR (issue number & link)
Issue: [#216](https://github.com/atlp-rwanda/atlp-devpulse-fn/issues/216)

### Screenshots (If appropriate especially on the frontend)
N/A
